auth/api/iam/api.go

```diff
@@ -412,7 +428,53 @@ func (r Wrapper) GetRequestJWT(ctx context.Context, request GetRequestJWTRequest
 // Extension of OpenID 4 Verifiable Presentations (OpenID4VP) on
 // RFC9101: The OAuth 2.0 Authorization Framework: JWT-Secured Authorization Request (JAR).
 func (r Wrapper) PostRequestJWT(ctx context.Context, request PostRequestJWTRequestObject) (PostRequestJWTResponseObject, error) {
-	return nil, errors.New("not implemented")
+	ro := new(jarRequest)
+	err := r.authzRequestObjectStore().Get(request.Id, ro)
```

Comment:
The request ID should be burned, to avoid client requesting it multiple times? (and causing load due to signing)